_posts/blog/2017-05-31-conversational-interfaces.md

```diff
@@ -0,0 +1,54 @@
+ 
+The act of conversation has the inherent side effect of being more personal and intimate. After all, it is the natural human reaction to personify things that speak (didn’t you enjoy the talking clock and candlestick in Disney’s _Beauty and the Beast_?). Use this to your advantage and set the tone, mood, and style to pervade the experience. 
+ 
+Start with relatable language. Conversation is intuitive and you shouldn’t have to “train” people how to use your product. Do not strive for the automated teller parallel (i.e “say ‘help’ if you need help, say ‘next’ if you want to hear more options”). Rather, be an expert in your specific area and direct the conversation accordingly. When you create the right persona, users won’t expect an all-knowing computer, just a proficient personality that is an expert in her field.
+
+## Be Concise 
+
+Unlike a GUI, where users can read and decide on options at their own pace, once a verbal response is given, users must rely on their memory of the spoken response to understand the next step — be concise and structure your requests for easy recall. Additionally, skimming a paragraph isn’t an option with a voice interface, which will frustrate a user who wants to advance quickly — to prevent this, be concise with your responses and remember foremost to answer the question the user asked. The whole experience must be easier than the alternative, or your users won’t be around for long.
+ 
+## Prevent Frustrating Moments
+
+Sometimes there are tough moments when users either didn’t say anything, or their response was misunderstood. Users have less tolerance for error in voice interfaces, because the root cause can be less obvious (you _know_ you made a mistake in a GUI when you fat-fingered something!) — how do you recover gracefully from these situations? 
+ 
+Acknowledge the ambiguity as soon as possible, since it will be more difficult to unravel later. Attempt to handle the correction using normal conversation. This can be accomplished by building a list of response prompts and randomizing them. Simply restating the exact question will come off as rude and impatient. Additionally, consider breaking the question into parts, or helping the user understand why you need more information. Understanding their progress through the experience will make them feel more comfortable answering the questions.
+ 
+Finally, as a general rule, if you have unsuccessfully asked the question three times (because presumably you must have the information to continue), exit the experience gracefully and hope it goes more smoothly next time.
+
+
+## Design a Great Experience
+
+After listening to several experts in the conversation UI field last week, I felt like all the messages boiled down to one maxim: 
+ 
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;**Always imagine the whole experience you are creating as a conversation between two people.**
```

Comment:
See above, this indent seems like an error. I think bold is enough to call it out without an indent. Other suggestions?